bark_ml/library_wrappers/lib_tf2rl/generate_expert_trajectories.py

```diff
@@ -418,6 +419,7 @@ def main_function(argv: list):
     param_servers = create_parameter_servers_for_scenarios(
         map_file, tracks_dir)
 
+    # TODO maybe make this a cli flag
     sim_time_step = 200
```

Comment:
As the merging blueprint is hardcoded for a timestep of 0.2s, I think it makes sense to fix it to 200ms on our side too. Maybe we can make it a command line flag, so we could test different values and later we could also change it in the blueprint.